app/index.html

```diff
@@ -101,13 +101,16 @@
     });
   </script>
 
+  <!-- Theme CSS - only populated by legacy VEDA-config instances, not used in standalone VEDA UI mode -->
+  <link rel="stylesheet" href="../static/styles/theme.css">
```

Comment:
oh it looks like parcel adds cache busting hash anyway even if the css was included in this way. noice